### PR TITLE
The gift button to allow users to send gifts to owners of a video in TC2

### PR DESCRIPTION
**Pull request by realswat1**
In this PR , I am implementing the gift button that would get the user_id of the playing stream and create a gift using the videos.user_Id as the receiver_id and that the sender_id from the verification. 
Next wallet amount for each user would be displayed on their profile page. 
  test plan :
https://www.loom.com/share/1a198bad0eef467da746231f8338109e?sid=56b267c3-9543-4f8a-9ef7-3f000b302db2

### CODE REVIEW

File: exposee-ui/src/components/Mainpage/Main.jsx [hunk 1]
```diff
@@ -74,13 +112,18 @@ function Main() {
...
             <div
-              className={`video-container ${
-                index === currentIndex ? "active" : ""
-              }`}
+              className={`video-container ${index === currentIndex ? "active" : ""}`}
               key={video.id}
             >
               <VideoPlayer playbackId={video.mux_playback_id} />
               <p>{video.title}</p>
+              {/* Add the gift button */}
```
Comment (Rajeshwari21) [inline exposee-ui/src/components/Mainpage/Main.jsx:120]: this is already done right? i see Gift button in the video so.

Comment (realswat1) [reply exposee-ui/src/components/Mainpage/Main.jsx:120]: it helps my reviewer what is coming next (ie. adding the gift button to the videos )

**Comment by Rajeshwari21** [comment]
Nice Job on this.

**Review by alfonsocheng — APPROVED** [review]
lgtm!

**Pull request merged**